apps/web/src/components/full-review/full-review.tsx

```diff
@@ -37,12 +37,15 @@ export const FullReview = ({ review, language }: FullReviewProps) => {
   const userProfileHref = `/${language}/${username}`
 
   return (
-    <div className="flex space-x-4">
-      <Link href={href} className="w-2/6 md:w-1/6">
+    <Link
+      href={`${href}?review=${review.id}`}
```

Comment:
Is there a specific reason not to concatenate the review id in the `const href `and just use `href={href}`?

Reply:
Another point is that the specific review link should only be in the title and on the poster, because when you click on see likes, for example, it opens the modal correctly but starts redirecting to the page of the item that was reviewed.

Reply:
> Is there a specific reason not to concatenate the review id in the `const href` and just use `href={href}`?

In href declaration I'd have to repeat the ´review.id´ twice, using like this I only inform it once.

Reply:
> Another point is that the specific review link should only be in the title and on the poster, because when you click on see likes, for example, it opens the modal correctly but starts redirecting to the page of the item that was reviewed.

I think it makes sense to only have one single link, if you go to the review page its already the movie's page. 

About the like button, maybe we could disabled it in this component and let the user only see the like list in the reviews page, what about it @lui7henrique ?

Reply:
About disabling the like button, I don't think it makes sense because it's kind of strange, it has the same component with two different behaviors.

And another thing, the link being the entire component, you can't select the review text if you want to copy it, or anything else text.

Example: https://letterboxd.com/films/

Reply:
> About disabling the like button, I don't think it makes sense because it's kind of strange, it has the same component with two different behaviors.
> 
> And another thing, the link being the entire component, you can't select the review text if you want to copy it, or anything else text.
> 
> Example: https://letterboxd.com/films/

Maybe we could add a "view" button instead of the component being a link?